### PR TITLE
Add compatibility for developing across different operating systems

### PR DESCRIPTION
**Pull request by mbkupfer**
Old npm scripts would crash on windows os (at least for windows 7) due
to use of single quotes and the non-compatible method of setting
environment variables. Former issue was resolved by adding double quotes
that were escaped with backslashes. The latter has been resolved by
using npm package "cross-env".

### CODE REVIEW

**Comment by mbkupfer** [comment]
@Gmousse sorry for the confusion with the two branches. I was developing from to computers and ended up confusing myself and only then did I realized what I did :laughing: 

Please continue using the latest fork that is on the develop branch and ignore my-develop

**Comment by mbkupfer** [comment]
@Gmousse , I believe it is now all ready to go. 

I feel that all these commits could fit in one squashed PR. Do you prefer that I squash on my own branch and resubmit, or do you want to do it?

File: src/reusables.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ export function loadTextFile(file, func) {
 }
 
 export function addFileProtocol(path) {
-    return path.startsWith("/") || path.startsWith("./")
+    return path.startsWith("/") || path.startsWith("./") || path.startsWith("C")
```
Comment (Gmousse) [inline src/reusables.js]: I forgot this code ! 
It's a good fix, but I think I will change this code part later because it's really too specific.

**Comment by Gmousse** [comment]
@mbkupfer Thank you.

Don't worry I will squash myself.

I will merge after my work.

File: src/reusables.js (hunk 1 above)
Comment (mbkupfer) [reply src/reusables.js]: No problem thanks for implementing the changes. Not sure what you had in mind for new version, but one suggestion is getting the root from [path.parse.](https://nodejs.org/api/path.html#path_path_parse_path)

Looking forward to seeing the new version!

Comment (Gmousse) [reply src/reusables.js]: I can't touch this part until a breaking 2.0.0.
Indeed the use of path.parse could be critical for the browser usage of the library.

In a 2.0.0 I hope to split browser and serveur usage and thus clarify this point.

**Pull request merged**